libs/types/src/tallies.ts

```diff
@@ -59,8 +59,10 @@ export interface ExternalTally {
   readonly numberOfBallotsCounted: number;
 }
 
+// Why have a source type with only one value? We used to support importing
+// external tallies from state election management systems and had more source
+// types.
 export enum ExternalTallySourceType {
-  SEMS = 'sems',
   Manual = 'manual-data',
 }
 
```

Comment:
Currently, all of the code for external tallies is generalized (since we had different external tally source types). Now, the only supported external tallies are manual. We could go change all of that code to be specific to manual tallies (e.g. changing the language used, removing the checking of the source type), which would probably make things a bit clearer.

I'm not sure if it's worthwhile for two reasons:
- I'm not sure how much clearer it would be
- I'm not sure if we'll want to support importing other types of external tallies anytime soon

Reply:
I think this is something we should do when we move that data to the backend.